charts/osm/README.md.gotmpl

```diff
@@ -0,0 +1,61 @@
+# Open Service Mesh Helm Chart
+
+![Version: 0.6.1](https://img.shields.io/badge/Version-0.6.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.6.1](https://img.shields.io/badge/AppVersion-v0.6.1-informational?style=flat-square)
+
+A Helm chart to install the [OSM](https://github.com/openservicemesh/osm) control plane on Kubernetes.
+
+## Prerequisites
+
+- Kubernetes v1.15+
+
+## Get Repo Info
+
+```console
+helm repo add osm https://openservicemesh.github.io/osm
+helm repo update
+```
+
+## Install Chart
+
+```console
+helm install [RELEASE_NAME] osm/osm
```

Comment:
Should this be `charts/osm`?

Reply:
No, it seems that's how it looks in the chart repo:

```
~> helm search repo osm
NAME   	CHART VERSION	APP VERSION	DESCRIPTION
osm/osm	0.6.1        	v0.6.1     	A Helm chart to install the OSM control plane o...
```